python/ray/actor.py

```diff
@@ -56,10 +54,15 @@ def bar(self):
         num_returns: The number of object refs that should be returned by
             invocations of this actor method.
     """
-    assert len(args) == 0
-    assert len(kwargs) == 1
-
-    assert "num_returns" in kwargs or "concurrency_group" in kwargs
+    valid_kwargs = ["num_returns", "concurrency_group"]
+    error_string = (
+        "The @ray.method decorator must be applied using at least one of "
+        f"the arguments in the list {valid_kwargs}, for example "
+        "'@ray.method(num_returns=2)'."
+    )
+    assert len(args) == 0 and len(kwargs) > 0, error_string
```

Comment:
Is it "at least one of" or "exactly one of" -- looks like the old behavior was exactly one?

Reply:
Right, I suspect that might be a bug. It isn't clear to me why `concurrency_group` and `num_returns` wouldn't be compatible with each other (for example, ray.remote doesn't have this limitation).

@jovany-wang do you have any context here? Looks like this was introduced in #18567. Old check was `assert len(kwargs) == 1` but doesn't seem clear why we can't have both here

Reply:
It's indeed a bug. I forgot to update the check in the old PR.

It should be `1 or 2`.